baqend/top100Testing.ts

```diff
@@ -116,20 +116,60 @@ function getFormattedResults(multiComparisons: model.BulkTest[]) {
     }
 
     formattedResult += `${multiComparison.url}\t`;
-    const fmpAverage = multiComparison.factors.firstMeaningfulPaint.toFixed(2).replace('.', ',');
-    const spAverage = multiComparison.factors.speedIndex.toFixed(2).replace('.', ',');
-    formattedResult += `${fmpAverage}\t`;
-    formattedResult += `${spAverage}\t`;
-
-    const fmpUplift = Math.round(multiComparison.competitorMeanValues.firstMeaningfulPaint - multiComparison.speedKitMeanValues.firstMeaningfulPaint);
-    const spUplift = Math.round(multiComparison.competitorMeanValues.speedIndex - multiComparison.speedKitMeanValues.speedIndex);
-    formattedResult += `${fmpUplift}\t`;
-    formattedResult += `${spUplift}\t`;
-
-    const fmpCompetitor = Math.round(multiComparison.competitorMeanValues.firstMeaningfulPaint);
-    const fmpSpeedKit = Math.round(multiComparison.speedKitMeanValues.firstMeaningfulPaint);
-    formattedResult += `${fmpCompetitor}\t`;
-    formattedResult += `${fmpSpeedKit}\n`;
+
+    // FMP Median Competitor
+    const fmpMedianCompetitor = multiComparison.testOverviews.sort((testOverview1, testOverview2) => {
```

Comment:
You don't need to compute the median for the metrics and test result separately. The comparison we are interested in is the overall second best comparison judging by the difference in FMP of the competitor and Speed Kit  test.